app/constants.py

```diff
@@ -98,6 +98,10 @@
 NOTIFICATION_STATUS_LETTER_RECEIVED = "received"
 DVLA_RESPONSE_STATUS_SENT = "Sent"
 
+# DVLA letter callback status
```

Comment:
could these be wrapped like `NOTIFICATION_STATUS_TYPES = []`? ideally these should have been enums

Reply:
addressed on the comment above: https://github.com/alphagov/notifications-api/pull/4157/commits/cab242e899a314bdbeb289b9ffa84a7c90ad5234